app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = 'User created'
+      redirect_to login_path
```

Comment:
- [ ] **[OPTIONAL]** We can sign in the user directly after signing up successfully instead of sending the user to the login page.